ivy/functional/ivy/experimental/activations.py

```diff
@@ -1,4 +1,5 @@
 # global
+from typing import Union, Optional, Literal
 from typing import Union, Optional, Callable, Literal
```

Comment:
I think something must have gone wrong here (possibly my fault with the merge)
```suggestion
from typing import Union, Optional, Callable, Literal
```